sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Release History
+
+## 1.0.0-beta.1
```

Comment:
The other one is beta2, why is this beta1?

Reply:
because those features were added in before beta1